app/pages/UcsfClientEligibilityPage/UcsfClientEligibilityPage.module.scss

```diff
@@ -0,0 +1,108 @@
+@import '~styles/utils/_helpers.scss';
+
+.eligibilityPage {
+  grid-column: 2 / 13;
+  @media screen and (max-width: 1370px) {
+    grid-column: 2/11;
+  }
+
+  @media screen and (max-width: $break-tablet-p) {
+    grid-column: 2/12;
+  }
+
+  @media screen and (max-width: $break-tablet-s) {
+    grid-column: 1/13;
+  }
+}
+
+.eligibilitiesContainer {
+  display: grid;
+  grid-template-columns: repeat(22, 1fr);
+  gap: 35px;
+
+  @media screen and (max-width: 1370px) {
+    grid-template-columns: 1fr;
+  }
+}
+
+.subtitleMargin {
+  margin-bottom: 13px;
+}
+
+.eligibilitiesBox {
+  grid-column: 1 / 16;
+  display: flex;
+  margin-top: 13px;
+  padding: 32px;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1), 0px 0px 4px rgba(0, 0, 0, 0.1);
+
+  @media screen and (max-width: 1370px) {
+    grid-column: 1;
+  }
+
+  @media screen and (max-width: $break-tablet-s) {
+    flex-direction: column;
+    align-items: center;
+  }
+
+  &_title {
+    color: $color-brand;
+    @extend %font-size-large;
+    width: 86px;
+    margin-right: 46px;
+    @media screen and (max-width: $break-tablet-s) {
+      width: auto;
+      margin: auto auto 15px;
+    }
+  }
+}
+
+.eligibilitiesList {
+  display: grid;
+  gap: 8px;
+}
+
+.eligibilitiesLabels {
+  display: grid;
+  gap: 45px;
+  list-style: decimal;
+}
+
+.eligibilityListItem {
+  display: list-item;
+}
+
+.eligibilityLabel {
+  cursor: pointer;
+  vertical-align: top;
```

Comment:
Hmm, I don't have a quick fix for this, but if you set the Checkbox's vertical-align to `middle` instead of `bottom` like it currently is, then you can also set the `vertical-align` for this element to `middle`, which will actually middle-align both of them.

Vertically aligning inline content is pretty tricky in general due to Western languages having descenders (e.g. the tail of a lowercase "g"). You probably don't need to read this right now, but I thought [this article](https://christopheraue.net/design/vertical-align) was one of the best at explaining how vertical alignment actually works for inline content, and the short summary is that it's really, really complicated and depends on the vertical alignment of all other items on the same line.

Once I spent a long time fiddling with a custom checkbox and vertical alignment of a label next to it, and I think what I ended up doing was keeping the default vertical alignment of `baseline` for both the checkbox and the label, but instead carefully positioning the checkbox vertically such that the baseline of checkbox was not at the bottom edge but instead slightly higher up. I also set the checkbox height and width in `em` so that it scales with the font size, which is important in terms of lining up the descender height of a typical Latin font with the checkbox's lower edge. The other really tricky thing is that this is also dependent on your font, since different fonts have different descender, x, and topline heights.

Anyway, I don't think we need to really change this now, but I wanted to point out a potential refactoring we may want to do in the future.